paparazzi-gradle-plugin/src/main/java/app/cash/paparazzi/gradle/PaparazziPlugin.kt

```diff
@@ -62,6 +63,10 @@ class PaparazziPlugin : Plugin<Project> {
         project.tasks.named("compile${testVariantSlug}Kotlin")
             .configure { it.dependsOn(writeResourcesTask) }
       }
+
+      project.tasks.named("test${testVariantSlug}", Test::class.java).configure {
+        it.systemProperties["PAPARAZZI_RESOURCES_DETAILS_FILE_KEY"] = "${project.buildDir}/${paparazziResourcesDetailsFile}"
```

Comment:
Another option I was considering, is to just generate that `resources.txt` file in, well, as resources and have the Paparazzi rule know exactly where to look for that file. That option will remove the need for this `systemProperties` addition and will still work on other build-system.

Reply:
it's a generated build/temp artifact.  moving it out would likely impose setting gitignores in consuming repos, which would be slightly annoying.

Reply:
I didn't mean to generate it and store it under `src` folder (so git would see it). Basically, write that file into the `build` folder and add that to the resources. But, this is for future work, not in this PR.